tests/ui/UnreadIndicatorsTest.js

```diff
@@ -43,6 +43,7 @@ jest.mock('react-native/Libraries/LogBox/LogBox', () => ({
 jest.mock('react-native-reanimated', () => ({
     ...jest.requireActual('react-native-reanimated/mock'),
     createAnimatedPropAdapter: jest.fn,
+    useReducedMotion: jest.fn,
```

Comment:
FYI these missing mocks will be added to Reanimated in https://github.com/software-mansion/react-native-reanimated/pull/5372